stable/velero/templates/cleanup-crds.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.cleanUpCRDs }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: velero-cleanup
```

Comment:
The name should use the fullname template

Reply:
Done.